webservices/common/models/filings.py

```diff
@@ -74,18 +78,34 @@ class EFilings(db.Model):
     coverage_end_date = db.Column('through_date', db.Date, doc=docs.COVERAGE_END_DATE)
     beginning_image_number = db.Column('starting', db.BigInteger, doc=docs.BEGINNING_IMAGE_NUMBER)
     ending_image_number = db.Column('ending', db.BigInteger, doc=docs.ENDING_IMAGE_NUMBER)
-    report_type = db.Column('rptcode', db.String, doc=docs.REPORT_TYPE)
+    report_type = db.Column('rptcode', db.String, db.ForeignKey(ReportType.report_type), doc=docs.REPORT_TYPE)
     # double check amendment interpretation
     amended_by = db.Column('superceded', db.BigInteger, doc=docs.AMENDED_BY)
     amends_file = db.Column('previd', db.BigInteger, doc=docs.AMENDS_FILE)
     amendment_number = db.Column('rptnum', db.Integer, doc=docs.AMENDMENT_NUMBER)
 
+    report = db.relationship(ReportType)
+
+    @property
+    def document_description(self):
+        return utils.document_description(
+            self.coverage_end_date.year,
+            clean_report_type(self.report.report_type_full),
+            None,
+            self.form_type,
+        )
+
     @property
     def is_amended(self):
         if self.superceded is not None:
             return True
         return False
 
+    @property
+    def pdf_url(self):
+        image_number = str(self.beginning_image_number)
+        return 'http://docquery.fec.gov/pdf/{0}/{1}/{2}.pdf'.format(image_number[-3:], image_number, image_number)
```

Comment:
These URLs are interesting.  But one thing I'd suggest is just use image number once, i.e. 
`return 'http://docquery.fec.gov/pdf/{0}/{1}/{1}.pdf'.format(image_number[-3:], image_number)`

But that's nitpicking really.